From 362d98b5ac6cda428f26615d0cd1f16aa0e4a35f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?F=C3=A1bio=20Rodrigues=20Ribeiro?= <farribeiro@gmail.com>
Date: Thu, 9 Jun 2022 18:55:11 -0300
Subject: [PATCH] Update Cargo.toml

---
 src-tauri/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src-tauri/Cargo.toml b/src-tauri/Cargo.toml
index 7468f93..8b14a40 100644
--- a/src-tauri/Cargo.toml
+++ b/src-tauri/Cargo.toml
@@ -15,7 +15,7 @@ tauri-build = { version = "1.0.0-beta.3" }
 [dependencies]
 serde_json = "1.0"
 serde = { version = "1.0", features = [ "derive" ] }
-tauri = { version = "1.0.0-beta.8", features = ["system-tray"] }
+tauri = { version = "1.0.0-beta.8", features = ["system-tray", "gtk-tray"] }
 
 [features]
 default = [ "custom-protocol" ]
